src/screens/novel/NovelScreen.tsx

```
@@ -71,7 +71,7 @@ const Novel = ({ route, navigation }: NovelScreenProps) => {
     lastRead,
     novelSettings: {
       sort = defaultChapterSort,
-      filter = '',
+      filter = 'AND hidden=0',
```

by default hide hidden chapters